source/jormungandr/jormungandr/interfaces/v1/StatedResource.py

```diff
@@ -61,3 +62,15 @@ def wrapper(*args, **kwargs):
                 register_used_coverages([region])
             return result
         return wrapper
+
+    def dispatch_request(self, *args, **kwargs):
```

Comment:
I think we can use: https://github.com/flask-restful/flask-restful/pull/532/files

modifying  `self` seems dangerous